app/assets/stylesheets/pregnancies.scss

```diff
@@ -16,8 +16,12 @@ $font-size: 17px;
   }
 }
 
-#menu {
-  border-right: 1px solid black;
+#sections {
+  border-left: 1px solid black;
+  padding-left: 15%;
```

Comment:
Can you pop out this padding-left? I don't want to remix this too heavy without getting approval from team ux, and I know they're in the process of undertaking a review of everything.